winterfell/webapp/src/js/claimConfirmCtrl.js

```diff
@@ -1,87 +1,87 @@
 var app = angular.module('vetafiApp');
-app.controller('claimConfirmCtrl', ['$scope', '$state', '$stateParams', 'net', 'claimService', 'Profile', 'vaService', '$uibModal',
-  function($scope, $state, $stateParams, net, claimService, Profile, vaService, $uibModal) {
+app.controller('claimConfirmCtrl', ['$scope', '$state', '$stateParams', 'net', '$uibModal', 'userValues', 'forms', 'formTemplateService',
+  'user', 'vaService',
+  function($scope, $state, $stateParams, net, $uibModal, userValues, forms, formTemplateService, user, vaService) {
     $scope.claimId = $stateParams.claimId;
-    $scope.user = Profile.getUser().user;
     $scope.vaAddress = vaService.getAddress();
-
-    $scope.mailingList = [
-      $scope.vaAddress,
-      // TODO: placeholder home address for now
-      {
-        title: 'Home',
-        address: {
-          sendTo: $scope.user.firstname + ' ' + $scope.user.lastname,
-          line1: '1223 4th Street',
-          line2: 'Apt #211',
-          city: 'San Francisco',
-          state: 'CA',
-          country: 'USA',
-          zip: '94158'
-        }
-      },
-    ];
-
+    console.log(user.user);
+    $scope.userAddress = user.user.contact.address;
+    $scope.userEmail = user.user.email;
     $scope.emailList = [
       {
         name: 'Me',
-        email: $scope.user.email
+        email: user.email
       }
     ];
 
-    // list of forms user has filled out
-    // TODO: placeholder forms for now
-    $scope.formsList = [
-      {
-        id: 'va_2723',
-        title: 'VA-27343 Intent to File',
-        summary: 'Something that blah blah holds your place in line when filing.'
-      }
-    ];
+    $scope.formsList = forms;
+    $scope.formsInfo = formTemplateService;
 
-    $scope.onClickAddAddress = function() {
-      return openModifyContactModal('address', $scope.mailingList);
-    };
+    console.log($scope);
 
-    $scope.onClickEditAddress = function(index) {
```

Comment:
thanks for fixing this part. We should only edit address/email (and it should reflect on the form?) and not be adding more addresses/emails.